Bottom: b5058ce8e86744331d78c831592e48543d77c44c
Top:    dd5f19c0295fb93ae7e18f16643b62bb5dccef76
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2009-04-28 13:38:05 -0400

Refresh of mpc5200-dma-split

---

diff --git a/sound/soc/fsl/mpc5200_dma.c b/sound/soc/fsl/mpc5200_dma.c
index 64255eb..edfdd0b 100644
--- a/sound/soc/fsl/mpc5200_dma.c
+++ b/sound/soc/fsl/mpc5200_dma.c
@@ -154,7 +154,6 @@ static int pcm_dma_psc_free(struct snd_pcm_substream *substream,
 	snd_pcm_set_runtime_buffer(substream, NULL);
 	return 0;
 }
-EXPORT_SYMBOL_GPL(mpc5200_audio_dma_hw_free);
 
 
 /**
@@ -163,7 +162,7 @@ EXPORT_SYMBOL_GPL(mpc5200_audio_dma_hw_free);
  * This function is called by ALSA to start, stop, pause, and resume the DMA
  * transfer of data.
  */
-static int pcm_dma_psc_trigger(struct snd_pcm_substream *substream, int cmd,
+static int psc_dma_pcm_trigger(struct snd_pcm_substream *substream, int cmd,
 			   struct snd_soc_dai *dai)
 {
 	struct snd_soc_pcm_runtime *rtd = substream->private_data;
@@ -275,7 +274,6 @@ static int pcm_dma_psc_trigger(struct snd_pcm_substream *substream, int cmd,
 
 	return 0;
 }
-EXPORT_SYMBOL_GPL(mpc5200_audio_dma_trigger);
 
 
 /**
